package.json

```diff
@@ -198,9 +203,11 @@
     "@types/bignumber.js": "^5.0.0",
     "@types/bs58check": "^2.1.0",
     "@types/circular-dependency-plugin": "^5.0.5",
+    "@types/cors": "^2.8.13",
     "@types/d3-array": "^3.0.1",
     "@types/dompurify": "^2.3.2",
     "@types/eth-url-parser": "^1.0.0",
+    "@types/express-serve-static-core": "^4.17.34",
```

Comment:
Confirmed looks weird without a matching package but is actually correct, this exposes some express internal types which are not exposed by `@types/express`